Library/Homebrew/formula.rb

```diff
@@ -1810,6 +1818,17 @@ def mirror(val)
       define_method(type) { |val| stable.send(type, val) }
     end
 
+    # @!attribute [w] gpg
+    # An optional mechanism to verify the download with GnuPG as well as the
+    # standard checksum verification. Upstream must publish signature files
+    # for us to be able to use this mechanism. Those signature files often
+    # have either a ".gpg" or ".sig" file extension.
+    #
+    # <pre>gpg "https://ftpmirror.gnu.org/wget/wget-1.18.tar.xz.sig"</pre>
+    def gpg(val)
+      stable.gpg(val)
+    end
```

Comment:
A checksum is pretty redundant, IMO. If the signature changes `gpg --verify` breaks. If the tarball/zipball/whatever changes, `gpg --verify` breaks. I can't see an attack method that an extra SHA256 solves.